docs/contexts/sidetree/sidetree-v0.0.md

```diff
@@ -0,0 +1,9 @@
+# Sidetree Context
+
+- [Latest JSON-LD Context](./sidetree-v0.0.jsonld)
+
+### Terminology
+
+<h4 id="usage"><a href="#usage">usage</a></h4>
```

Comment:
this link points to itself

Reply:
its supposed to its for building an anchor to the definition.

Reply:
for example:

http://element-did.com/contexts/sidetree/sidetree-v0.0.md#usage

this is part of jsonld.